### ISSUE TITLE
Error: Could not connect to device, handshake timeout

### ISSUE THREAD

**Issue by leoneleone** [nicoh88/homebridge-xiaomi-roborock-vacuum#9]
@nicoh88 

Hello Nico. 
I'd like some help with another homebridge plugin which relies on miio for device connection and control. Sorry for posting an issue here, but that's the only way I could a message to you.

My issue:
1. I have managed to get a valid/working token for my Xiaomi device and can connect and control the device via the latest version of miio when using cli.
2. When running homebridge with the hoomebridge-xiaomi-humidifier. This plugin relies on miio being installed globally, which goes against the advice of the miio creator. I suspect that might be the source of the problem.
https://github.com/acc-ua/homebridge-xiaomi-humidifier
the plugin initializes but I keep getting an connection error in homebridge
https://github.com/acc-ua/homebridge-xiaomi-humidifier/issues/12

So basically I can connect and control via miio cli, but not viathe homebridge plugin mentioned above.

While I realize this isn't an issue with your plugin, I'm hoping you can help me fix this issue as you seem far more knowledgeable than I am.

Your insight would be much appreciated 🙂

**Comment by nicoh88** [comment]
Sorry, too many projects, too little time.

**Comment by leoneleone** [comment]
No worries. I understand.
